stable/rabbitmq/README.md

```diff
@@ -78,10 +78,19 @@ The following table lists the configurable parameters of the RabbitMQ chart and
 | `rabbitmq.configuration`             | Required cluster configuration                   | See values.yaml                                         |
 | `rabbitmq.extraConfiguration`        | Extra configuration to add to rabbitmq.conf      | See values.yaml                                         |
 | `rabbitmq.advancedConfiguration`     | Extra configuration (in classic format) to add to advanced.config    | See values.yaml                                         |
+| `rabbitmq.tls.enabled`                  | Enable ssl support to rabbitmq |   | `false`                     |
+| `rabbitmq.tls.failIfNoPeerCert`     | When set to true, TLS connection will be rejected if client fails to provide a certificate  | `true` |
+| `rabbitmq.tls.sslOptionsVerify`     | `verify_peer`  | Should [peer verification](https://www.rabbitmq.com/ssl.html#peer-verification) be enabled? |
+| `rabbitmq.tls.caCertificate`     | Ca certificate  | Certificate Authority (CA) bundle content |
+| `rabbitmq.tls.serverCertificate`     | Server certificate  | Server certificate content |
+| `rabbitmq.tls.serverKey`     | Server Key  | Server private key content |
+| `rabbitmq.tls.existingSecret`                                   | Existing secret with certificate content to rabbitmq credentials                                                                                                                  | `nil`                                                    |
 | `service.type`                       | Kubernetes Service type                          | `ClusterIP`                                             |
 | `service.port`                       | Amqp port                                        | `5672`                                                  |
+| `service.SSLPort`                   | Amqp ssl port                                    | `5671`                                                  |
```

Comment:
Could we rename this parameter to `service.tlsPort`?

Reply:
sure, renamed